pkg/ovs/openflow/interfaces.go

```diff
@@ -216,6 +216,7 @@ type FlowBuilder interface {
 	MatchCTStateTrk(isSet bool) FlowBuilder
 	MatchCTStateInv(isSet bool) FlowBuilder
 	MatchCTMark(value uint32) FlowBuilder
+	MatchCTLabelRange(high, low uint64, rng Range) FlowBuilder
```

Comment:
nit: I think `bitRange` would be a better name everywhere you use `rng`

Reply:
Done.